content/md/products-super-power/enrich-your-reference-entity-records.md

```diff
@@ -0,0 +1,142 @@
+---
+id: enrich-your-reference-entity-records
+themes: products-super-power, manage-your-reference-entities
+title: "Enrich your reference entity records"
+popular: false
+ee-only: right
+related:
+---
+
+# Enrich your reference entity records
+
+## Add a record
+Once you’ve created a reference entity, you can add **records**.
+Let’s say that you have just created the reference entity `Brand`, you have defined its records attributes and now it is time to add new records. For example: Fermob, Kartell, Muuto, Fatboy…
```

Comment:
Let’s say that you just have created
Put "just" before "have" to insist on the "recently" timing, otherwise, it can be seen as "only".

Reply:
For example: Fermob, Kartell, Muuto, Fatboy…
These brands are not really known, it's usually better to use more popular brands, like Nike, Adidas, Reebok...